### PR TITLE
Fix mobile balance modal colors

### PR DESCRIPTION
**Pull request by joel-jeremy**
Fixes https://github.com/actualbudget/actual/issues/3489

### CODE REVIEW

**Comment by youngcw** [comment]
This also exists in edge, but maybe you could fix it here.  The carryover arrow is no longer in line.
image

**Comment by joel-jeremy** [comment]
Adding new mobile VRTs as part of this PR. Hopefully this stabilizes some of the mobile VRTs a bit.

**Comment by joel-jeremy** [comment]
> The colors all look fixed! But found a different issue. This also exists in edge, but maybe you could fix it here. The carryover arrow is no longer in line. image

I'll just handle this as part of a separate PR :)

**Comment by youngcw** [comment]
How hard would it be to split out the new vrt to its own PR?  I would like to get this fix in soon, but the vrt changes probably would take a bit to get working and reviewed.

**Comment by joel-jeremy** [comment]
Separated mobile VRTs to https://github.com/actualbudget/actual/pull/3505

File: packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx [hunk 1]
```diff
@@ -102,101 +111,113 @@ export function BalanceWithCarryover({
...
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+          textAlign: 'right',
+          ...(!isDisabled && {
+            cursor: 'pointer',
+          }),
+          ':hover': { textDecoration: 'underline' },
+        }),
+      ),
+    [getBalanceAmountStyle, isDisabled],
```
[the remark this answers is not in the document]
Comment (joel-jeremy) [reply packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx:136]: glamor's css function returns a StyleAttribute, not a string

**Comment by youngcw** [comment]
Found a bug.
image


Recreate:
1. make a category with a template and then run the template
2. adjust the page so that the templated category balance is right underneath the "X" to close the balance modal when viewed
image

3. close the balance modal

**Review by youngcw — APPROVED** [review]

**Pull request merged**